src/Sylius/Bundle/AddressingBundle/spec/Form/EventListener/BuildAddressFormSubscriberSpec.php

```diff
@@ -34,12 +36,12 @@ function let(ObjectRepository $countryRepository, FormFactoryInterface $factory)
 
     function it_is_initializable()
     {
-        $this->shouldHaveType('Sylius\Bundle\AddressingBundle\Form\EventListener\BuildAddressFormSubscriber');
+        $this->shouldHaveType(BuildAddressFormSubscriber::class);
```

Comment:
Hey,

I'm not sure what you mean by `BuildAddressFormSubscriber` not existing. It does?

I see what you mean about the check being useless. I'll revert all similar `it_is_initializable` tests.
Are you sure the `implements` checks are also useless?